app/controllers/index.js

```diff
@@ -1,7 +1,85 @@
 import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
 
 export default class IndexController extends Controller {
   sort = '-created';
   page = 0;
   size = 20;
+
+  @service store;
+  @service router;
+
+  @tracked showModal = false;
+  @tracked hasBeenFocused = false;
```

Comment:
This is tracked but not used in the template?

Is it possible to use the value showModel for this instead of a new one?